samples/infix/writers.cc

```diff
@@ -358,6 +513,42 @@ namespace
       return false;
     }
 
+    // These 2 outputs below are a postfix encoding of variadic operations,
```

Comment:
I think if you pushed the size first it would be closer to what a typical postfix approach would be.

Reply:
I don't think that works. The size has to be where it is, as the `tuple` operator has to pop it first unconditionally. Putting it at the other end means the `tuple` has to already know how many elements it needs to pop to get to it.

Realistically, the prose is actually a little smarter and explains that Forth dialects usually have a special syntax for tuples rather than this counting stuff, but as-is I think this is the only way round it can work.